### DISCUSSION TITLE
Agents (Masks)?

### DISCUSSION THREAD

**Discussion by MarkAC007** [ChatGPTNextWeb/NextChat#2967] [Ideas]
I'll start by saying a thank you for this project, it's great!  It would be great if we could invoke 2 masks at the same time and have them interact between each other.   Eg: agent type functionality.

**Comment by H0llyW00dzZ** [comment]
I used based of openai fine-tuning as "systems"
https://github.com/H0llyW00dzZ/ChatGPT-Next-Web/commit/dc9694e1705868cd46281a249b21d00974860577
Screenshot :
![Screenshot_251](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/202ca066-c91a-475a-a4a1-4a112d256cf1)

![Screenshot_254](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/2be2bf92-8996-4d12-bbd9-bd1251c9945f)

also when you tryna do fine-tuning for custom models it's barely got a training loss

![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/dfe38977-d5c7-4c59-9d50-ab65d4a25420)
Tools : https://github.com/H0llyW00dzZ/OpenAI-API

edited :
combine 2 systems (role) its perfect
ref : https://openai.com/blog/gpt-3-5-turbo-fine-tuning-and-api-updates

**Comment by MarkAC007** [comment]
Ah….  Nice thinking.  I have not played with fine-tuning yet.  Will try this today.   Thank you

Best Regards

Mark Almeida-Cardy

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
